documentation/docs/Exercises/Exercise 5.md

```diff
@@ -33,50 +33,34 @@ Axelar Network is a work in progress. At no point in time should you transfer an
 
 Follow the instructions in [Setup with Docker](/setup-docker) or [Setup with Binaries](/setup-binaries) to make sure your node is up to date and you received some test coins to your account.
 
-
 ### Sending tokens from Axelar Network to Ethereum
 
-The testnet currently supports:
-- `satoshi`
-- `uphoton`(ibc/287EE075B7AADDEB240AFE74FA2108CDACA50A7CCD013FA4C1FCD142AFA9CA9A)
-- `uaxl`
-
-:::note
-[key-name] mentioned in this exercise can be the key name you used in Exercise 3.
-The address associated with the key name is funded, and you suppose to have some `satoshi` and `uphoton`.
-:::
+In what follows we assume your Axelar account name is `validator`.  This is the default name for the account that is automatically created for you when you first joined the Axelar testnet.
 
 1. Check that you have balances on your account
+
 ```bash
-axelard q bank balances $(axelard keys show [key-name] -a)
+axelard q bank balances $(axelard keys show validator -a)
 ```
 You should see your balances shows e.g.,
 ```bash
 balances:
-- amount: "10000"
- denom: satoshi
 - amount: "1000000"
  denom: uaxl
-- amount: "1000000"
- denom: ibc/287EE075B7AADDEB240AFE74FA2108CDACA50A7CCD013FA4C1FCD142AFA9CA9A
 ```
 2. Create a deposit address on Axelar Network (to which you'll deposit coins later)
 
-[token] is one of `uaxl`, `satoshi`, and `uphoton`
+[token] is `uaxl`.
```

Comment:
Should we add Terra uusd token? I think it replaced uphoton

Reply:
> Should we add Terra uusd token? I think it replaced uphoton

Makes sense to me.  I just did what I was told. 😛 @haiyizxx any opinion here?

Reply:
ya, fine to remove. We do UST from Terra to EVM directly in exercise 4 now, exercise 5 is for AXL from Axelar to EVM.